pdns/pdnsutil.cc

```diff
@@ -391,12 +391,14 @@ int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, const vect
 
     content.str("");
     content<<rr.qname<<" "<<rr.qtype.getName()<<" "<<rr.content;
-    if (recordcontents.count(toLower(content.str()))) {
+    string contentstr = content.str();
+    if (rr.qtype.getCode() != QType::TXT) contentstr=toLower(contentstr);
```

Comment:
this indentation / no newline took me three times to read what was going on

Reply:
will verbosify